crm_deduplicate_acl/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Deduplicate Contacts ACL
+========================
+
+This module extends the functionality of the CRM contact deduplicator to add
+permission groups that allow the matching users to use those tools, not
+needing to be the sale settings manager.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Settings > Users > Users*.
+#. Choose a user.
+#. Choose the desired permission level in *Appplication > Deduplicate
+   Contacts*:
+
+   - *Manually* allows him to do the manual deduplication process.
+   - *Automatically* allows him to do the automatic deduplication process.
+
+     .. warning::
```

Comment:
I hope they fix the theme some day. In the mean time it writes "Warning", should be enough don't you think?

![captura de pantalla de 2016-02-19 16-43-18](https://cloud.githubusercontent.com/assets/973709/13180255/f99f84d8-d727-11e5-80a4-5a730802f0c2.png)